### PR TITLE
Fix compilation errors

### PR DESCRIPTION
**Pull request by willdunklin**
Tried compiling the project on Ubuntu 20.04/gcc and ran into some simple compilation errors

### CODE REVIEW

File: src/engine/common/math.hpp [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #pragma once
 #include <algorithm>
+#include <math.h>
```
Comment (johnBuffer) [inline src/engine/common/math.hpp]: Maybe `<cmath>` would be better here?

**Review by johnBuffer — COMMENTED** [review]
Thank you very much for the fix, I just have a small suggestion :)

File: src/engine/common/math.hpp (hunk 1 above)
Comment (willdunklin) [reply src/engine/common/math.hpp]: It definitely works, however I do get warnings about `narrowing conversion of ‘sin(((double)((float)angle)))’ from ‘double’ to ‘float’`. It does still solve the compilation error though so works for me :)

Comment (johnBuffer) [reply src/engine/common/math.hpp]: In that case let's go for it! I will take care of the warnings :)
Thanks again for the fix!

**Pull request merged**